### ISSUE TITLE
PATH not set on LinuxCNC RPi4 image after installing LitexCNC

### ISSUE THREAD

**Issue by Peter-van-Tol** [Peter-van-Tol/LiteX-CNC#74]
Following the documentation for installing LitexCNC / LinuxCNC on a RaspberryPi4 using the image from linuxcnc.org/downloads does not work. 

Problems:
- LitexCNC is not persisted on `PATH`
- therefore the toolchain is also not on `PATH`, thus building fails.

**Proposed solution**
Modify the documentation so the PATH variable is persisted. In later stage modify source code so the relevant files are correctly written.

**Caveat**
The user _uba100_ on forum.linuxcnc.org has tried to fix this problem. He changed the `PATH` variable
``` shell
export PATH=$PATH:~/.local/bin/:~/toolchain/oss-cad-suite/libexec/
```
And got the following message:
```
INFO:SoC:
yosys: error while loading shared libraries: libffi.so.7: cannot open shared object file: No such file or directory
```

This shared file is located within `oss-cad-suite`, so he changed the `LD_LIBRARY_PATH` to include this folder:
``` shell
export LD_LIBRARY_PATH=~/toolchain/oss-cad-suite/lib/:$LD_LIBRARY_PATH
```
Which now results in receiving the following messages:
```
/home/cnc/.local/bin/python3: /home/cnc/toolchain/oss-cad-suite/lib/libm.so.6: version `GLIBC_2.35' not found (required by /home/cnc/.local/bin/python3)
/home/cnc/.local/bin/python3: /home/cnc/toolchain/oss-cad-suite/lib/libc.so.6: version `GLIBC_2.33' not found (required by /home/cnc/.local/bin/python3)
/home/cnc/.local/bin/python3: /home/cnc/toolchain/oss-cad-suite/lib/libc.so.6: version `GLIBC_2.32' not found (required by /home/cnc/.local/bin/python3)
/home/cnc/.local/bin/python3: /home/cnc/toolchain/oss-cad-suite/lib/libc.so.6: version `GLIBC_2.34' not found (required by /home/cnc/.local/bin/python3)
```

This most likely indicates a broken installation. Changing `LD_LIBRARY_PATH` is most-likely not the correct way to get the installation up-and-running.

**Comment by Peter-van-Tol** [comment]
Problem is that `PATH` variable is not set correctly. The parameters are written to `.profile`, which is not read by the console when started from the GUI. I tested it headless over SSH, which means that `.profile` is read...

Temporary fix for the problem is to add the following lines to `.bashrc`
```
# set PATH so it includes user's private bin if it exists
if [ -d "$HOME/.local/bin" ] ; then
    PATH="$HOME/.local/bin:$PATH"
fi
# set PATH so it includes oss-cad-suite
if [ -d "/home/cnc/toolchain/oss-cad-suite/bin" ] ; then
        PATH="/home/cnc/toolchain/oss-cad-suite/bin:$PATH"
fi
```

**Comment by Peter-van-Tol** [comment]
Solved in version 1.2.1
